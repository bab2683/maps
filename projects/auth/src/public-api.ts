/*
 * Public API Surface of auth
 */

export * from './lib/auth.model';
export * from './lib/auth.module';
export * from './lib/store';
