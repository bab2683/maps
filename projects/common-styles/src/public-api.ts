/*
 * Public API Surface of common-styles
 */
export * from './lib/common-styles.module';
