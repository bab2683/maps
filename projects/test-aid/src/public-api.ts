/*
 * Public API Surface of test-aid
 */

export * from './lib/angular-fire.mock';
export * from './lib/rxjs.mock';
export * from './lib/utils.mock';
