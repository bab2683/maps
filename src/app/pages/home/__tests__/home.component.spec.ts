import { NO_ERRORS_SCHEMA } from '@angular/compiler/src/core';
import { ComponentFixture, TestBed } from '@angular/core/testing';

import { HomeComponent } from '../home.component';

describe('HomeComponent', () => {
  let component: HomeComponent;
  let fixture: ComponentFixture<HomeComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [HomeComponent],
      schemas: [NO_ERRORS_SCHEMA]
    }).compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(HomeComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
