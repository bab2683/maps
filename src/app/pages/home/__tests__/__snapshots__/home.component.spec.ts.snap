// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should match snapshot 1`] = `
<app-home>
  <h1>
    Home page
  </h1>
</app-home>
`;
